Update README.md (#11)
crystal-lang-tools/tree-sitter: README.md

> # Crystal Treesitter

Reviewer: ```suggestion
# tree-sitter-crystal

```

I'm not sure this should be changed as this is how the other tree-sitter implementations have the header in the README

> Hello! You're looking at an ongoing WIP [tree-sitter](https://github.com/tree-sitter/tree-sitter) language support project ;) for the [Crystal](https://crystal-lang.org/) language. **Why?** This is a part of a drive to increase Crystal tooling across workflows (myself using neovim), and specifically consolidate efforts in the Crystal Tree Sitter realm. See [this post](https://forum.crystal-lang.org/t/tree-sitter-crystal/3565) for more context on this repos creation.

Reviewer: ```suggestion
This is a WIP implementation of [tree-sitter](https://github.com/tree-sitter/tree-sitter) for the [Crystal](https://crystal-lang.org) programming language. This is part of a drive to increase Crystal tooling across workflows and IDEs, and specifically to consolidate efforts to create a tree-sitter implementation for Crystal. See [this forum post](https://forum.crystal-lang.org//t/tree-sitter-crystal/3565) for more context on this repos creation.
```

> ## Installation

Reviewer: ```suggestion
```

An installation guide can come later when the project is in a more stable state, there's not much people can use this for right now.

> ___

Reviewer: ```suggestion
```

> ## How to Contribute

Reviewer: ```suggestion
## Contributing

```

For a general contribution guideline, see [this repo](https://github.com/MarcDiethelm/contributing/tree/master). 

> To communicate with maintainers, please utilize this repo & GitHub's social sections.

Reviewer: ```suggestion
```

> ### To-Do

Reviewer: ```suggestion

### Current Status

```

For an updated to-do list, see the [pinned issue](https://github.com/crystal-lang-tools/tree-sitter/issues/2).

> ___

Reviewer: ```suggestion
```

> ## Thanks & Inspirations

Reviewer: ```suggestion
## Thanks & Inspirations

```
